reference-architecture/rhv-ansible/ocp-vars.yaml

```diff
@@ -45,6 +45,7 @@ root_ssh_key:
 deployment_type: openshift-enterprise
 openshift_vers: v3_5
 containerized: false
+console_port: 8443
```

Comment:
Re-reading, I'd misunderstood... I think a default value would require a role structure. Can you do that for a playbook?